cmd/getroughtime/main.go

```diff
@@ -1,4 +1,16 @@
-// Copyright 2018 Cloudflare, Inc.
+// Copyright 2023 Cloudflare, Inc.
```

Comment:
you could use a range: `2018-2023`

Reply:
Reverted to 2018 since there are no significant changes here.